### DISCUSSION TITLE
How to change tekton pipeline and task for karavan

### DISCUSSION THREAD

**Discussion by via1982** [apache/camel-karavan#732] [General]
Hi,

I'm trying to deploy my test-spring-boot app via karavan dashboard and after push the deploy button face the issue below.
As I understand I should change karavan-task-dev-spring-boot.yaml in part of scripts sections in order to work with bitbucket correctly.
Am I correct? How can I do this?

step-karavan-build-deploy
Cloning into '/scripts'...
fatal: could not read Password for 'https://password@bitbucket.XXX.com': No such device or address
/tekton/scripts/script-0-7cz27: line 15: cd: /scripts/testilya: No such file or directory
jbang -Dcamel.jbang.version=3.20.3 camel@apache/camel export --local-kamelet-dir=/scripts/kamelets
The runtime option must be specified
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
[INFO] Processing build on daemon be851798
[INFO] Scanning for projects...

**Comment by mgubaidullin** [comment]
1. Get existing task from kubernetes using `kubectl get tasks.tekton.dev karavan-task-dev-spring-boot -n karavan -o yaml >> task.yaml`
2. Edit `script` part of `task.yaml` 
3. Apply new version to kubernetes using 'kubectl apply -f task.yaml`

**Comment by mgubaidullin** [comment]
https://github.com/apache/camel-karavan/issues/805

**Comment by davsclaus** [comment]
closing old tickets
